src/js/tabs/servos.js

```diff
@@ -41,49 +41,45 @@ TABS.servos.initialize = function (callback) {
         let servoCheckbox = '';
         let servoHeader = '';
         for (let i = 0; i < FC.RC.active_channels-4; i++) {
-            servoHeader = servoHeader + '\
-                <th >A' + (i+1) + '</th>\
-            ';
+            servoHeader += `<th>A${(i+1)}</th>`;
```

Comment:
Has the servo functionality been tested?

Reply:
@TheIsotopes has tested this? Did you test it with an actual servo motor?